mmv1/products/secretmanager/SecretVersion.yaml

```diff
@@ -130,13 +145,29 @@ properties:
     description: The secret payload of the SecretVersion.
     required: true
     custom_flatten: 'templates/terraform/custom_flatten/secret_version_access.go.tmpl'
+    custom_expand: 'templates/terraform/custom_expand/secret_version_access.go.tmpl'
     flatten_object: true
     properties:
-      - name: 'secret_data'
+      - name: 'secretData'
         type: String
         description: The secret data. Must be no larger than 64KiB.
         api_name: data
-        required: true
+        conflicts:
+          - 'secretDataWo'
         immutable: true
         sensitive: true
-        custom_expand: 'templates/terraform/custom_expand/secret_version_secret_data.go.tmpl'
+      - name: 'secretDataWo'
+        type: String
+        description: The secret data. Must be no larger than 64KiB. For more info see [updating write-only attributes](/docs/providers/google/guides/using_write_only_attributes.html#updating-write-only-attributes)
+        api_name: data
+        required_with:
+          - 'SecretDataWoVersion'
+        conflicts:
+          - 'payload.0.secretData'
+        write_only: true
+      - name: 'SecretDataWoVersion'
```

Comment:
These should all be `url_param_only: true`, right? This one is `ignore_read: true`

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/13135/commits/6c9d5245c5f2e95090876fa316d90557c7fd994c